integreat_cms/cms/migrations/0084_language_socialmedia_webapp_title.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.22 on 2023-12-07 15:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Add translatable socialmedia title
+    """
+
+    dependencies = [
+        ("cms", "0083_add_page_based_offer_fields"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="language",
+            name="socialmedia_webapp_title",
+            field=models.CharField(
+                default="Integreat | Web-App | Lokale Informationen für Dich",
```

Comment:
Quite sure, this does not work correctly for malte & aschaffenburg. Should I omit the default value? Also, we actually already have translations for this in the frontend, which could be "ported" to the backend somehow(?/by hand?).

Reply:
We should do this migration relatively soonish to merging this pr then.